intg/src/main/java/org/apache/atlas/model/instance/AtlasRelationship.java

```diff
@@ -71,6 +72,10 @@ public class AtlasRelationship extends AtlasStruct implements Serializable {
     public static final String KEY_BLOCKED_PROPAGATED_CLASSIFICATIONS = "blockedPropagatedClassifications";
     public static final String KEY_PROPAGATED_CLASSIFICATIONS         = "propagatedClassifications";
 
+    // RELATIONSHIP_DEF and RELATIONSHIP_END_TO_ES_DOC_ID_MAP used for Kafka consumers to handle ES relationship sync
+    public static final String RELATIONSHIP_DEF = "relationshipDef";
+    public static final String RELATIONSHIP_END_TO_ES_DOC_ID_MAP = "relationshipEndToESDocIdMap";
```

Comment:
Sample values in this map? "Relationship_end" means relationship guid or something else?

Reply:
Sample value: 
`"relationshipEndToESDocIdMap": {
      "end1DocId": "3rxc",
      "end2DocId": "3lo0"
}`